lib/soapy_cake/client.rb

```diff
@@ -39,6 +33,21 @@ def run(request)
 
     private
 
+    def fetch_opt(key, fallback = nil)
+      opts.fetch(key, ENV.fetch("CAKE_#{key.to_s.upcase}", fallback))
```

Comment:
True, but at least they're all isolated to this `Client` class.  It made the code a bit simpler (per rubocop's request) and enforces that the option and ENV variable are named consistently.